mixer/pkg/cache/cache_test.go

```diff
@@ -142,7 +154,7 @@ func testCacheConcurrent(c Cache, t *testing.T) {
 		t.Errorf("Got %d hits, expecting %d", stats.Hits, workers*numIters)
 	}
 
-	if stats.Writes != uint64(workers*numIters*2) {
```

Comment:
Not sure why this changed ?

Reply:
Cause eviction isn't counted as a write.